javascript/javascript-4.md

```diff
@@ -88,3 +89,18 @@ The output should look similar to this:
 `Erin is willing to go to Banff, Alberta, Canada or Iceland.`  
 `Janet is willing to go to London or Hogwarts.`  
 
+### Check for Understanding
+
+- Differentiate between for loop and while loop?
+- Write down the syntax of Nested for loop in JavaScript?
+- Write a program in JavaScript to print Multiplication table of given number using for loop. Also try the program using while loop.
```

Comment:
Can you move this to line 91 and make it:
### Challenge
- Write a program in JavaScript to print Multiplication table of given number using for loop. Also try the program using while loop.